script/fixes/105_modaction_to_postevents.rb

```diff
@@ -62,3 +78,7 @@ def create(post_id, creator_id, created_at, action, extra_data = {})
     end
   end
 end
+
+ActiveRecord::Base.connection.execute('INSERT INTO post_events (creator_id, post_id, "action", extra_data, created_at)
```

Comment:
This needs a without timeout block around it, or it's likely to fail in production. The prod server has fairly strict global statement timeouts applied to it.